js/sprite.js

```diff
@@ -10,14 +10,34 @@ import { Sprite, Game, Scene } from 'athenajs';
 const myScene = new Scene(),
     mySprite = new Sprite();
 
-mySprite.addAnimation('runningMan', 'img/axeBandit.png', {
-    numFrames: 8,
-    frameWidth: 80,
-    frameHeight: 80,
-    frameDuration: 4
+mySprite.addAnimation('jump', 'img/sprites.png', {
+    numFrames: 10,
+    frameWidth: 60,
+    frameHeight: 72,
+    frameDuration: 2,
+    offsetY: 72
 });
+
+mySprite.addAnimation('run', 'img/sprites.png', {
+    numFrames: 10,
+    frameWidth: 60,
+    frameHeight: 72,
+    frameDuration: 2
+});
+
+mySprite.setAnimation('run');
+
 myScene.addObject(mySprite);
 
+// mySprite.animate('Rotate', {
+//     startValue: 0,
+//     endValue: Math.PI * 2,
+//     duration: 1000,
+//     loop: 2
+// });
+
+window.mySprite = mySprite;
```

Comment:
implicit mutation of global var `window` ? shouldn't it be exported then be used ?